src/main/java/javaserver/Methods/Handler.java

```diff
@@ -0,0 +1,19 @@
+package javaserver;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class Handler {
```

Comment:
It seems like you never actually instantiate this class. I would look into making this an interface or an abstract class